common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,154 @@
+              Then I should not see the update "Hello"
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.update_text_contains('Hello'))
+        self.course_updates_page.click_delete_update_button()
+        self.assertFalse(self.course_updates_page.update_text_contains('Hello'))
+
+    def test_user_edit_update(self):
+        """
+        Scenario: Users can edit update dates
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            And I add a new update with the text "Hello"
+            When I edit the date to "06/01/13"
+            Then I should see the date "June 1, 2013"
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.update_text_contains('Hello'))
+        self.course_updates_page.click_edit_update_button()
+        self.course_updates_page.set_date('06/01/2013')
+        self.course_updates_page.click_new_update_save_button()
+        self.assertTrue(self.course_updates_page.is_update_date('June 1, 2013'))
+
+    def test_outside_tag_preserved(self):
+        """
+        Scenario: Text outside of tags is preserved
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            When I add a new update with the text "before <strong>middle</strong> after"
+            Then I should see the update "before <strong>middle</strong> after"
+            And when I reload the page
+            Then I should see the update "before <strong>middle</strong> after"
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('before <strong>middle</strong> after')
+        self.assertTrue(self.course_updates_page.update_text_contains('before <strong>middle</strong> after'))
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.update_text_contains('before <strong>middle</strong> after'))
+
+    def test_asset_change_in_updates(self):
+        """
+        Scenario: Static links are rewritten when previewing a course update
+           Given I have opened a new course in Studio
+           And I go to the course updates page
+           When I add a new update with the text "<img src='/static/my_img.jpg'/>"
+           # Can only do partial text matches because of the quotes with in quotes (and regexp step matching).
```

Comment:
This is a pretty weak test, but since it was already this way, I'm OK with it staying as-is.